### DISCUSSION TITLE
Collapsable sections

### DISCUSSION THREAD

**Discussion by ruslansteiger** [statamic/cms#5403] [General]
Hey 👋

My blueprints get massive sometimes, and it would be cool to have something like collapsable sections. I made quick mockup how it could look like:

<img width="1254" alt="Screenshot 2022-03-03 at 19 50 43" src="https://user-images.githubusercontent.com/14092921/156632681-54d790c7-1161-44cf-9873-b67d15afa03e.png">

Anyone interested in this feature? If there is interest, I would like to contribute to Statamic and make a pull request. 🙏

**Comment by jasonvarga** [comment]
This is what the tabs are designed for.

Create multiple sections in your blueprint rather than using the section fieldtype, which is basically just a heading.

**Reply by ruslansteiger** [reply]
Tabs are great. In my new project I have 15 different sections and each section has multiple fields (2-10). It is pretty confusing with tabs and to edit the content is not that intuitive.

Example Screenshot of the live preview with tabs:

<img width="988" alt="image" src="https://user-images.githubusercontent.com/14092921/156635847-6dd80a08-1419-4403-84ae-68aaf7a66bcc.png">

Currently, I solve that problem with a repeater and just do a single set in it. With this solution it is pretty easy to follow the flow of the document and edit stuff. But it is kinda annoying.

<img width="497" alt="image" src="https://user-images.githubusercontent.com/14092921/156636199-18822948-4610-4303-aefb-fa8f200753cf.png">

**Reply by jasonvarga** [reply]
It'd be a pretty huge change to support what you're asking.

Or, maybe if you wanted to somehow nest the fields within the section, that could potentially be done. This is already sorta requested at:
- https://github.com/statamic/ideas/issues/210
- https://github.com/statamic/ideas/issues/638

Edit: See Jack's response. There are a couple of addons that do this already!

**Comment by edalzell** [comment]
I like the idea, but would prefer it be a double click or a drop arrow and not a toggle

**Comment by jackmcdade** [comment]
You could try one of these addons that are quite similar to what you're looking to do!

- https://statamic.com/addons/andrew-haine/accordion-group
- https://statamic.com/addons/goldnead/collapse-fieldtype

**Reply by ruslansteiger** [reply]
Thank you @jackmcdade it works like a charm 😘

Next time I should check the addons page before posting here ✌️

**Comment by jackmcdade** [comment]
Awesome!
